lib/ProductOpener/Attributes.pm

```diff
@@ -1405,6 +1404,9 @@ sub compute_attribute_ingredients_analysis($$$) {
 	# the ingredients_analysis taxonomy contains en:palm-oil and not en:contains-palm-oil
 	$analysis_tag =~ s/contains-(.*)$/$1/;
 
+	# Link to the corresponding knowledge panel (the panel id depends on the value of the property)
+	$attribute_ref->{panel_id} = "ingredients_analysis_en:" . $analysis_tag;
```

Comment:
There is also a panel for environmental impact, with id panel_palm_oil
![Capture d’écran de 2022-05-24 10-23-17](https://user-images.githubusercontent.com/144788/169985004-637a8448-7654-4efa-851c-c3ce6bc175d3.png)

I don't know which is best to link to…

Reply:
For the panel linked to the palm oil attribute, I think the ingredients analysis panel is the best one, as it explains why we think the product contains or may contain palm oil. One thing we could do is to add the Threatened species panel has a subpanel of the palm oil ingredients analysis panel.

Reply:
Filed https://github.com/openfoodfacts/openfoodfacts-server/issues/6820